pkgs/by-name/co/copilot-language-server/package.nix

```diff
@@ -1,31 +1,52 @@
 {
   lib,
-  buildNpmPackage,
-  fetchurl,
+  stdenvNoCC,
+  fetchzip,
+  nix-update-script,
 }:
 
-buildNpmPackage rec {
+let
+  arch =
+    {
+      aarch64-darwin = "arm64";
+      aarch64-linux = "arm64";
+      x86_64-darwin = "x64";
+      x86_64-linux = "x64";
+    }
+    ."${stdenvNoCC.hostPlatform.system}"
+      or (throw "Unsupported system: ${stdenvNoCC.hostPlatform.system}");
+  os =
+    {
+      aarch64-darwin = "darwin";
+      aarch64-linux = "linux";
+      x86_64-darwin = "darwin";
+      x86_64-linux = "linux";
+    }
+    ."${stdenvNoCC.hostPlatform.system}"
+      or (throw "Unsupported system: ${stdenvNoCC.hostPlatform.system}");
+in
+
+stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "copilot-language-server";
   version = "1.275.0";
 
-  src = fetchurl {
-    url = "https://registry.npmjs.org/@github/copilot-language-server/-/copilot-language-server-${version}.tgz";
-    hash = "sha256-OVqtwz9T5vSYAZc8nof0jXn7H40i1r7SAS6jK4xeSlo=";
+  src = fetchzip {
+    url = "https://github.com/github/copilot-language-server-release/releases/download/${finalAttrs.version}/copilot-language-server-native-${finalAttrs.version}.zip";
+    hash = "sha256-tygL6bXYRtcBI0iEO02HXHflp7XX3l9s+0Z1XRoMqms=";
```

Comment:
If you intend to update, you can use `0h6w1yclwm9ks7hpfspkzczrbs1kr4a23cak10mp6ixkj853hkrh` as the hash for version `1.286.0`.

Reply:
It seems this change is due to auto update, so rebased.